README.md

```diff
@@ -10,83 +10,159 @@
 Compile and distribute Python extensions written in Rust as easily as if
 they were written in C.
 
-## Setup
+## Quickstart
 
-For a complete example, see
-[html-py-ever](https://github.com/PyO3/setuptools-rust/tree/main/examples/html-py-ever).
+The following is a very basic tutorial that shows how to use `setuptools-rust` in `pyproject.toml`.
+It assumes that you already have a bunch of Python and Rust files that you want
+to distribute. You can see examples for these files in the 
+[`examples/hello-world`](https://github.com/PyO3/setuptools-rust/tree/main/examples/hello-world)
+directory in the [github repository](https://github.com/PyO3/setuptools-rust).
+The [PyO3 docs](https://pyo3.rs) have detailed information on how to write Python
```

Comment:
Should we also mention `rust-cpython` here? Although I don't think there is any example using `rust-cpython` (or is there?)

Reply:
I think it's fine as-is; `rust-cpython` is relatively unmaintained recently and as you say we also lack any example of it here. I think if someone wanted to contribute an example and extend documentation to discuss `rust-cpython` I would accept it, just no need to block this PR on it.